src/app/world-map/world-map.component.ts

```diff
@@ -0,0 +1,34 @@
+import { AfterViewInit, Component } from '@angular/core';
+import * as L from 'leaflet';
+
+@Component({
+  selector: 'app-world-map',
+  templateUrl: './world-map.component.html',
+  styleUrls: ['./world-map.component.scss'],
+})
+export class WorldMapComponent implements AfterViewInit {
+  private map;
+
+  constructor() {}
+
```

Comment:
Empty constructors can (and probably should) be omitted.
Which reminds me, we need to set up linting.